docs/quickstarts/aws.md

```diff
@@ -0,0 +1,207 @@
+
+To create a Lokomotive cluster, we need to define a configuration.
+
+A [production-ready configuration](../../examples/aws-production) is already provided for ease of
+use.  Copy the example configuration to the working directory and modify accordingly.
+
+The provided configuration installs the Lokomotive cluster and the following components:
+
+* [metrics-server](../configuration-guides/components/metrics-server.md)
+* [openebs-operator](../configuration-guides/components/openebs-operator.md)
+* [cert-manager](../configuration-guides/components/cert-manager.md)
+* [flatcar-linux-update-operator](../configuration-guides/components/flatcar-linux-upate-operator.md)
+* [openebs-storage-class](../configuration-guides/components/openebs-storage-class.md)
+* [prometheus-operator](../configuration-guides/components/prometheus-operator.md)
+
+You can configure the components as per the requirements.
+
+Create a variables file `lokocfg.vars` to define the variables present in the configuration file.
+
+```console
+#lokocfg.vars
+ssh_public_keys = ["public-ssh-key-1", "public-ssh-key-2", ...]
+
+state_s3_bucket = "name-of-the-s3-bucket-to-store-the-cluster-state"
+lock_dynamodb_table = "name-of-the-dynamodb-table-for-state-locking"
+
+dns_zone = "dns-zone-name"
+route53_zone_id = "zone-id-of-the-dns-zone"
+
+cert_manager_email = "email-address-used-for-cert-manager-component"
+grafana_admin_password = "password-for-grafana"
+```
+
+For advanced cluster configurations and more information refer to the [AWS configuration
+guide.](../configuration-guides/platforms/aws.md)
+
+### Step 5: Create Lokomotive cluster
+
+Run the following command to create the cluster:
+
+```console
+lokoctl cluster install
+```
+Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
+under `asset_dir`.
+
+## Verification
+
+A successful installation results in the output:
+
+```console
+module.aws-myawscluster.null_resource.bootkube-start: Still creating... [1m50s elapsed]
+module.aws-myawscluster.null_resource.bootkube-start: Still creating... [2m0s elapsed]
+module.aws-myawscluster.null_resource.bootkube-start: Creation complete after 2m6s [id=5156996152315868880]
+
+Apply complete! Resources: 118 added, 0 changed, 0 destroyed.
+
+Your configurations are stored in /home/imran/lokoctl-assets/myawscluster
+
+Now checking health and readiness of the cluster nodes ...
+
+Node              Ready    Reason          Message
+
+ip-10-0-39-75     True     KubeletReady    kubelet is posting ready status
+ip-10-0-39-78     True     KubeletReady    kubelet is posting ready status
+ip-10-0-39-29     True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-241    True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-244    True     KubeletReady    kubelet is posting ready status
+ip-10-0-12-249    True     KubeletReady    kubelet is posting ready status
+
+Success - cluster is healthy and nodes are ready!
```

Comment:
The output will be different, as also the components will be installed. I'm not sure if that matters though.